src/Components/Form.jsx

```diff
@@ -0,0 +1,163 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { ImInfo } from "react-icons/im";
+function Form() {
+  const [breed, setBreed] = useState("");
+  const [weight, setWeight] = useState("");
+  const [height, setHeight] = useState("");
+  const [isInfo, setIsInfo] = useState(false);
+  const [heightUnits, setHeightUnits] = useState("in");
+  const [weightUnits, setWeightUnits] = useState("lb");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(breed, weight, height, heightUnits, weightUnits);
+  };
+
+  const toggleWeight = (value) => {
+    setWeightUnits(value);
+  };
+
+  const toggleHeight = (value) => {
+    setHeightUnits(value);
+  };
+  return (
+    <FormDiv onSubmit={handleSubmit}>
+      <label>What is your dog's breed?</label>
+      <input onChange={(e) => setBreed(e.target.value)}></input>
+      <div className="container">
+        <label>What is your dog's weight?</label>
+        <button
+          className={weightUnits === "lb" ? "active" : ""}
+          onClick={() => toggleWeight("lb")}
+        >
+          lb
+        </button>
+        <button
+          className={weightUnits === "kg" ? "active" : ""}
+          onClick={() => toggleWeight("kg")}
+        >
+          kg
+        </button>
+      </div>
+      <input
+        className="shorten-input"
+        onChange={(e) => setWeight(e.target.value)}
+      ></input>
+      <div className="container">
+        <label>What is your dog's height?</label>
+        <StyledInfoButton onClick={() => setIsInfo(!isInfo)} />
+        <button
+          className={heightUnits === "in" ? "active" : ""}
+          onClick={() => toggleHeight("in")}
+        >
+          in
+        </button>
+        <button
+          className={heightUnits === "cm" ? "active" : ""}
+          onClick={() => toggleHeight("cm")}
+        >
+          cm
+        </button>
+      </div>
+      {isInfo && (
+        <div>
+          <p>
+            Measure from the ground to the highest point of the shoulderblade on
+            the back, and not the neck or head
+          </p>
+        </div>
+      )}
+      <input
+        className="shorten-input"
+        onChange={(e) => setHeight(e.target.value)}
+      ></input>
+      <button type="submit" className="submit">
+        Submit
+      </button>
+    </FormDiv>
+  );
+}
+
+const FormDiv = styled.form`
+  display: flex;
+  flex-direction: column;
+  border: 1px solid lightgray;
+  border-radius: 8px;
+  box-shadow: 0px 0px 10px lightgray;
+  height: 15rem;
+  width: 20rem;
+  padding: 1.5rem;
+  font-family: "Inter", sans-serif;
+  label {
+    margin-top: 0.5rem;
+    margin-bottom: 0.5rem;
+  }
+  .container {
+    display: flex;
+    align-items: center;
+    label {
+      margin-right: 0.3rem;
+    }
+    button {
+      font-family: "Inter", sans-serif;
```

Comment:
Same comment as above. Since the font-family isn't something we're changing a lot, we can reduce code duplication by adding it to global styles. It would also help to move any other styles that are default across the board.